packages/maskbook/src/components/InjectedComponents/SetupGuide.tsx

```diff
@@ -103,12 +105,34 @@ const useWizardDialogStyles = makeStyles((theme) =>
             border: `${theme.palette.type === 'dark' ? 'solid' : 'none'} 1px ${theme.palette.divider}`,
             overflow: 'hidden',
         },
+        rootMobile: {
```

Comment:
FYI. You can use `[theme.breakpoints.up('xs')]: {}` for mobile styling.

Reply:
ok